meinberlin/assets/scss/components_user_facing/_hero.scss

```diff
@@ -0,0 +1,104 @@
+.hero {
+  width: 100%;
+  background-color: $gray-lightest;
+}
+
+.hero__container {
+  max-width: 980px;
+  min-height: 352px;
+  margin: 0 auto;
+  position: relative;
+  display: flex;
+  flex-direction: column-reverse;
+}
+
+.hero__content {
+  width: 100%;
+  padding: 1rem;
+}
+
+.hero__title {
+  margin-top: 0;
+  margin-bottom: 10px;
+}
+
+.hero__title-link {
+  color: $text-base!important;
```

Comment:
Usually I increase specificity of the selector by nesting more, when trying to override BO styles. Would that be an option? Or do you need to override an id selector?